poleno/mail/tests/test_backend.py

```diff
@@ -121,56 +120,54 @@ def test_message_with_html_body_and_text_alternative(self):
         self.assertEqual(mail.instance.html, u'<p>HTML content</p>')
         self.assertEqual(list(mail.instance.attachment_set.all()), [])
 
-    @unittest.skip(u'FIXME')
     def test_message_with_text_body_and_multiple_alternatives(self):
         mail = self._send_email(body=u'Text content', alternatives=[
             (u'<p>HTML alternative 1</p>', u'text/html'),
-            (u'<p>HTML alternative 2</p>', u'text/html'),
+            (u'<html><body><p>HTML alternative 2</p></body></html>', u'text/html'),
```

Comment:
Ked uz ako sample davame skutocne HTML, tak ho davajme konzistentne vsade, kde je sample HTML dokument.

Aby ten sample dokument bol co najkratsi, tak `<p>` asi mozeme vyhodit. Staci `<html><body>HTML alternative 2</body></html>`

Reply:
:ok: Mozno by bolo vhodne to upravit aj v `poleno/utils/tests/`, tam sa to tiez pouziva.

Reply:
@viliambalaz Ano, mozes to zmenit aj v `poleno/utils/tests/`, nech vsade pouzivame rovnake sample pre HTML

Reply:
@viliambalaz Toto chces spravit v samostatnom PR, ci tu?

Reply:
@martinmacko47 Ano v samostatnom PR, nechcem miesat veci z `poleno/utils/` spolu s `poleno/mail`. Nazbieralo sami uz viacej veci na upravu. Tieto upravy spravim v jednom spolocnom novom PR.

Reply:
@viliambalaz Ok. Aby sme na to nezabudli, pls napis do issue komentar, co este chceme spravit v separatnych PR. Ked sa to spravi, tak ten komentar zedituj a oznac :heavy_check_mark: , ze to je hotove.

Sem potom daj linku na ten komentar, nech ked v buducnosti bude niekto prechadzat tento PR, tak nech si vie dohladat, ako tento thread skoncil.

Reply:
https://github.com/martinmacko47/chcemvediet/issues/280#issuecomment-697153280